modules/update-upgrading-cli.adoc

```diff
@@ -114,10 +124,10 @@ $ oc get clusterversion
 [source,terminal]
 ----
 
-Cluster version is <version> 
+Cluster version is <version>
 
 Upstream is unset, so the cluster will use an appropriate default.
-Channel: stable-4.10 (available channels: candidate-4.10, candidate-4.11, eus-4.10, fast-4.10, fast-4.11, stable-4.10) 
+Channel: stable-4.10 (available channels: candidate-4.10, candidate-4.11, eus-4.10, fast-4.10, fast-4.11, stable-4.10)
```

Comment:
A side question, do we intend to mention stable-4.10 in all 4.10+ docs? We have 4.9, 4.10 mentioned in current 4.12 doc. Do we plan to update the pages to be version agnostic?

Reply:
If example output such as this would still be useful to users even if the version numbers were replaced with some agnostic placeholder, I'd be in favor of doing that.

Using old examples like this in newer docs, and updating with newer examples every few versions, seems like an okay but less ideal alternative as long as the examples are still relevant/useful to users.

Updating these example outputs with technically accurate version numbers for every release seems like a maintainability headache, and this would be my least preferred option. But I'm still okay with doing this if that's what everyone else would like.

@LalatenduMohanty @wking any opinions?